inst/qml/auditClassicalModelFairness.qml

```diff
@@ -25,7 +25,7 @@ import "./common"	as Common
 Form 
 {
 	columns: 							1
-	info:								qsTr("The fairness metrics analysis enables the user to assess fairness and discrimination regarding specific groups in the data in algorithmic decision-making systems. Considering a certain positive class in the data, fairness -or discrimination- can be quantified using model-agnostic fairness metrics. The ratio of two fairness metrics is called parity, which is a well-known concept in algorithmic fairness.")
+	info:								qsTr("The fairness measures analysis enables the user to assess fairness and discrimination regarding specific groups in the data in algorithmic decision-making systems. Considering a certain positive class in the data, fairness -or discrimination- can be quantified using model-agnostic fairness metrics. ")
```

Comment:
There is an extra space at the end here. Also, why is `The ratio of two fairness metrics is called parity, which is a well-known concept in algorithmic fairness.` removed?